src/main/definitions/api/caseApiResponse.ts

```diff
@@ -65,6 +65,61 @@ export interface CaseData {
   claimServedDate?: string;
   genericTseApplicationCollection?: GenericTseApplicationTypeItem[];
   sendNotificationCollection?: SendNotificationTypeItem[];
+  hearingCollection?: HearingModel[];
+}
+
+export interface HearingModel {
+  id: string;
+  value: {
+    Est_Hearing?: {
+      fromDays: string;
+      fromHours: string;
+      fromMinues: string;
+    };
+    Hearing_stage?: string;
+    hearingFormat: string[];
+    Hearing_type?: string;
+    hearingNumber: string;
+    hearingSitAlone: string;
+    judicialMediation: string;
+    hearingEstLengthNum: number;
+    hearingPublicPrivate: string;
+    Hearing_notes?: string;
+    Hearing_date_start?: Date;
+    Hearing_judge_name?: string;
+    Hearing_venue?: {
+      value: HearingVenueItem;
+      list_items: HearingVenueItem[];
+      selectedCode: string;
+      selectedLabel: string;
+    };
+    hearingDateCollection: {
```

Comment:
this collection is mandatory? or it could be with question mark?

Reply:
I think the hearingDateCollection is mandatory. A hearing needs to have at least one date.